docs/source/quickstart.rst

```diff
@@ -1,22 +1,256 @@
+.. _Quickstart:
+
 **********
 Quickstart
 **********
 
-Under Construction
+Generate a new ingest package
+=============================
+
+The first thing to do after installing the ingest library CLI
+is to create a new package named ``my_study``::
+
+$ kidsfirst new --dest_dir=my_study
+
+We can run it and test to make sure everything works by using the following
+command
+
+.. code-block:: text
+
+    $ kidsfirst test my_study
+
+The logs on your screen should indicate a successful test run like this:
+
+.. code-block:: text
+
+    ======================================== 1 passed in 0.12s ============================================================
+    2020-02-18 17:40:36,039 - DataIngestPipeline - Thread: MainThread - INFO - ✅ User defined data validation tests passed
+    2020-02-18 17:40:36,039 - DataIngestPipeline - Thread: MainThread - INFO - END data ingestion
+    2020-02-18 17:40:36,039 - kf_lib_data_ingest.app.cli - Thread: MainThread - INFO - ✅ Ingest pipline passed validation!
+
+This package has a simple test piece of data to extract from. You need to write an additional extraction
```

Comment:
Remove these two lines, we'll introduce the extract configs in the next subsection. See below suggestion